<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>711</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_fetch">
     <attribute name="title">
      <string>Fetch</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="fetchButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Fetch your Mail</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTextEdit" name="fetchField">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Inbox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sent</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Send">
     <attribute name="title">
      <string>Send</string>
     </attribute>
     <widget class="QTextEdit" name="textField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>561</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="toField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="attachButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>40</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Attachement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Mail Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_server_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subjectField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_server_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Subject</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>561</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Attachments :</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Domains</string>
     </attribute>
     <widget class="QLineEdit" name="domainField">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>22</y>
        <width>197</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="smtp_portField">
      <property name="geometry">
       <rect>
        <x>368</x>
        <y>50</y>
        <width>196</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>110</y>
        <width>77</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Domain</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_imap_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>78</y>
        <width>84</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IMAP Server :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="smtpField">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>197</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="imap_portField">
      <property name="geometry">
       <rect>
        <x>368</x>
        <y>78</y>
        <width>196</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_port_2">
      <property name="geometry">
       <rect>
        <x>293</x>
        <y>78</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IMAP  Port :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imapField_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>78</y>
        <width>197</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_port_3">
      <property name="geometry">
       <rect>
        <x>293</x>
        <y>50</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SMTP Port :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_server_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>84</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SMTP Server :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>22</y>
        <width>84</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Domain :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>287</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>List of Domains :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="listButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>77</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>List Domains</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>571</width>
        <height>371</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>611</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Domain Name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="background">
        <color>
         <red>175</red>
         <green>231</green>
         <blue>230</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SMTP Server</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="background">
        <color>
         <red>175</red>
         <green>231</green>
         <blue>230</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SMTP Port</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="background">
        <color>
         <red>175</red>
         <green>231</green>
         <blue>230</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IMAP Server</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="background">
        <color>
         <red>175</red>
         <green>231</green>
         <blue>230</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IMAP Port</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="background">
        <color>
         <red>175</red>
         <green>231</green>
         <blue>230</blue>
        </color>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail Address :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailField">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>aymane--13@outlook.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pwd">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwdField">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>aout131995</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
